samples/matter/light_switch/src/app_task.cpp

```diff
@@ -319,3 +187,33 @@ void AppTask::UserTimerTimeoutCallback(k_timer *timer)
 
 	TaskExecutor::PostTask([timerType]() { TimerEventHandler(timerType); });
 }
+
+CHIP_ERROR AppTask::Init()
+{
+	/* Initialize Matter stack */
+	ReturnErrorOnFailure(Nordic::Matter::InitServer(MatterEventHandler));
+
+	/* Initialize application timers */
+	k_timer_init(&sDimmerPressKeyTimer, AppTask::UserTimerTimeoutCallback, nullptr);
+	k_timer_init(&sDimmerTimer, AppTask::UserTimerTimeoutCallback, nullptr);
+
+	LightSwitch::GetInstance().Init(kLightSwitchEndpointId);
```

Comment:
This initializes BindingHandler that accesses Server's members before `Server` is initialized.

Reply:
Right, this should be done in the mPostServerInitClbk.